src/tls/test-server.c

```diff
@@ -466,16 +464,10 @@ test_tls_client_cert_disabled (TestCase *tc, gconstpointer data)
 static void
 test_tls_client_cert_expired (TestCase *tc, gconstpointer data)
 {
-#if GNUTLS_VERSION_NUMBER >= 0x030604
-  /* GnuTLS 3.6.4 introduces TLS 1.3 by default, which has only a two-step
```

Comment:
Please leave this comment here.  It's the only thing that leaves me not scratching my head about "why the heck doesn't `handshake()` just fail?!"

Reply:
re-added, with some tweaking.